modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -37,22 +45,23 @@ The Endpoint Configuration window is displayed.
 +
 Your API is now created.
 
-[[create-api-ASM-CLI]]
-== Create an API Using Anypoint Service Mesh
+[[create-api-autocreate]]
+== Create an API Using the Anypoint Service Mesh Autocreate Feature
 
-Use the procedure in this section to automatically create an API using Anypoint Service Mesh. Before you create the API, you must complete the task prerequisite.
+When you configure the Anypoint Service Mesh autocreate feature, an API is automatically created for you. How you authenticate your user for the create transaction depends on whether your users are managed by Anypoint Platform or an external IdP.
 
-*Task Prerequisite*
+[[create-api-ASM-CLI]]
+=== Autocreate by Using Your User Credentials
 
-Before you begin, ensure that you obtain the:
+If your users are managed by Anypoint Platform, use the username and password credentials for authentication. Before you configure the Anypoint Service Mesh autocreate feature, ensure that you obtain the:
 
 * Environment ID and organization ID. 
 +
 To obtain these details, go to *API Manager > Environment Information*.
 * Manage APIs Configuration permission for the environment where you are creating the API Instance.
```

Comment:
```suggestion
* Manage APIs Configuration permission for the environment where you are creating the API instance
```
Is "Manage APIs Configuration" really the name of the permission?

Reply:
Yes! Done! Thanks!